crates/csskit_proc_macro/src/snapshots/csskit_proc_macro__test__bounded_range_multiplier_is_optimized_to_options.snap

```diff
@@ -22,8 +21,18 @@ impl<'a> ::css_parse::Parse<'a> for Foo {
     fn parse(p: &mut ::css_parse::Parser<'a>) -> ::css_parse::Result<Self> {
         use ::css_parse::{Parse, Peek};
         let val0 = p.parse::<crate::AnimateableFeature>()?;
-        let val1 = p.parse_if_peek::<crate::AnimateableFeature>()?;
-        let val2 = p.parse_if_peek::<crate::AnimateableFeature>()?;
+        let val1 = if p.peek::<crate::AnimateableFeature>() {
```

Comment:
Yep, since we rely on a _generic_ `parse_steps` method, this code looks a little funky, given `parse_if_peek`. But I think functionally they are equivalent. And should a specific parse_steps do more (say bounded Angles), we'll correctly handle those.